roles/barbican/defaults/main.yml

```diff
@@ -0,0 +1,58 @@
+---
+project_name: barbican
+barbican_keystone_auth: yes
+barbican:
+  admin_workers: 1
+  public_workers: 1
+  admin_project_name: admin
+  admin_project_domain_name: Default
+  uwsgi:
+    method: port
+    http_port:
+      admin: 9313
```

Comment:
There should be only one public uwsgi http_port for Barbican. Keystone has an admin and public endpoint but not Barbican.